client/src/components/exercise/lab9/pages/Repairs/DateRepair.js

```diff
@@ -1,30 +1,36 @@
 /* eslint-disable no-empty-pattern */
 import React from "react";
-import { PropTypes } from "prop-types";
+import PropTypes from "prop-types";
 import useRepairDate from "../../hooks/useRepairDate";
 import Repair from "../../../../body/Repair/Repair";
-import { GAME_STATES, REPAIR } from "../../../../../constants/lab9";
+import {
+  // eslint-disable-next-line no-unused-vars
+  REPAIR_DATE_REPAIR_HEADING,
+  GAME_STATES,
+  REPAIR,
+} from "../../../../../constants/lab9";
+
 /**
  * Date Repair is a component that is responsible for passing logic into the universal
```

Comment:
little thing but just get rid of the spacing between the two words and just do `DateRepair():` in the doc-string